ext/fg/js/frontend-initialize.js

```diff
@@ -24,49 +24,100 @@
  * apiOptionsGet
  */
 
+async function createIframePopupProxy(url) {
+    const rootPopupInformationPromise = yomichan.getTemporaryListenerResult(
+        chrome.runtime.onMessage,
+        ({action, params}, {resolve}) => {
+            if (action === 'rootPopupInformation') {
+                resolve(params);
+            }
+        }
+    );
+    apiBroadcastTab('rootPopupRequestInformationBroadcast');
+    const {popupId, frameId} = await rootPopupInformationPromise;
+
+    const frameOffsetForwarder = new FrameOffsetForwarder();
+    frameOffsetForwarder.start();
+    const getFrameOffset = frameOffsetForwarder.getOffset.bind(frameOffsetForwarder);
+
+    const popup = new PopupProxy(popupId, 0, null, frameId, url, getFrameOffset);
+    await popup.prepare();
+
+    return popup;
+}
+
+async function getOrCreatePopup(depth) {
+    const frameOffsetForwarder = new FrameOffsetForwarder();
```

Comment:
Two FrameOffsetForwarder instances can exist at the same time this way; one from `createIframePopupProxy` and one from `getOrCreatePopup`.

Reply:
https://github.com/FooSoft/yomichan/pull/439/commits/362ac84e6c6a0a4c2353232a5c44e1b2157de44d